.github/workflows/build_test.yaml

```diff
@@ -17,10 +17,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Checkout repository
```

Comment:
Checkout is ok. It is just checkout in other repos.

Reply:
Done